kibana/templates/configmap-dashboardimport.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.dashboardImport.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "kibana.fullname" . }}-importscript
+  labels:
+{{ include "kibana.labels" . | indent 4 }}
+data:
+  dashboardImport.sh: |
+    #!/usr/bin/env bash
+    #
+    # kibana dashboard import script
+    #
+
+    cd /kibanadashboards
+
+    echo "Starting Kibana..."
+
+    /usr/local/bin/kibana-docker $@ &
+
+    echo "Waiting up to {{ .Values.dashboardImport.timeout }} seconds for Kibana to get in green overall state..."
+
+    for i in {1..{{ .Values.dashboardImport.timeout }}}; do
+      curl -s localhost:5601/api/status | python -c 'import sys, json; print json.load(sys.stdin)["status"]["overall"]["state"]' 2> /dev/null | grep green > /dev/null && break || sleep 1
```

Comment:
I already commented on this in the values.yaml, but hard coding the protocol and port is going to break for anyone using https or a different port. The readinessProbe has a function that handles all of the different use cases.

https://github.com/elastic/helm-charts/blob/a3865369ab6a4a257453145a6d20b1d8266f6a94/kibana/templates/deployment.yaml#L94-L106

Anyone using a custom webroot will also have issues causing all of the api calls to fail.